src/bank/two/Transfer.java

```diff
@@ -24,8 +26,12 @@ public Account getDstAccount() {
     @Override
     public void execute() {
         if(!this.isExecuted){
-            this.srcAccount.reduceBalance(this.amount);
-            this.isExecuted = true;
+            try {
+                this.srcAccount.reduceBalance(this.amount);
```

Comment:
W "Clean Code" tego typu deklaracjom poświęca się stosunkowo dużo uwagi, mianowicie chodzi o to by nazewnictwo obiektów było możliwie najbardziej deskryptywne - zwiększa to czytelność kodu. Stosowanie skrótów nie jest zalecane. Dlatego lepszym rozwiązaniem byłby obiekt o nazwie sourceAccount.